wp-content/themes/blank/template-parts/slider-section.php

```diff
@@ -1,10 +1,10 @@
-<div class="full-landscape-section">
+<div class="slider-section-section">
     <!-- <div class="container" style="background-image: url(<?php the_sub_field('landscape_image');?>)"> -->
-    <div class="container" style="background-image: url(https://placeimg.com/1200/600/any)">
+    <div class="" style="background-image: url(https://www.charbonnel.co.uk/media/adminforms/image_carousel/cache/1/cache/5e06319eda06f020e43594a9c230972d/b/a/banner_1920x630-5_1a.jpg)">
```

Comment:
```suggestion
    <div style="background-image: url(https://www.charbonnel.co.uk/media/adminforms/image_carousel/cache/1/cache/5e06319eda06f020e43594a9c230972d/b/a/banner_1920x630-5_1a.jpg)">
```
remove empty class, also should you be linking to a cached file as this is not guaranteed to exist

Reply:
This will be removed after I have added ACF